pkgs/state-transition-runtime/src/main.zig

```diff
@@ -30,18 +33,31 @@ pub fn main() noreturn {
     };
     _ = witnesses;
```

Comment:
we still have not accessed inputs and witnesses here right?

Reply:
yes, this is coming in another PR that I'm currently working on, see blockblaz/zeam-pm#9